rules/Arguments/Rector/FuncCall/FunctionArgumentDefaultValueReplacerRector.php

```diff
@@ -15,9 +15,6 @@
 use Webmozart\Assert\Assert;
 
 /**
- * @changelog https://php.watch/versions/8.1/version_compare-operator-restrictions
```

Comment:
I think `@changelog` can be replaced with `@see`

Reply:
That's an option. Should be part of rule definition though, otherwise it's only visible when rule is opened (rarely :))

Reply:
E.g. this link is completelly wrong. It's a different feature.
That's why I want to remove those, as docblocks are not part of rule definition and can be incorrectly copy-pasted.

Reply:
Just checked couple of those and they're either:

* link to 3v4l.org = those should be in a code example snippet or tests
* links to PHP rfc
* dead links

I'll keep them removed, as I want to keep focused on sets rather than particular rules. Similar way PHPStan does, to make it easy to install Rector and run it :+1: